targets/pca10056.json

```diff
@@ -2,7 +2,6 @@
 	"inherits": ["nrf52840"],
 	"build-tags": ["pca10056"],
 	"flash-method": "command",
```

Comment:
Only pca10056 has command as the default.
It might be better to adapt it to openocd, but I don't know.
I will fix it if someone wants to.